ion/symboltable.go

```diff
@@ -281,19 +293,22 @@ func (s *bogusSST) String() string {
 
 	buf := strings.Builder{}
 	w := NewTextWriter(&buf)
-	w.Annotations(SymbolToken{Text: &ionSharedSymbolTableText, LocalSID: 9}, SymbolToken{Text: &bogusText, LocalSID: SymbolIDUnknown})
```

Comment:
Warnings, ha!? Yikes

Reply:
Yes all of this was just to make Goland stop flagging it.